test-support/trie-bench/Cargo.toml

```diff
@@ -14,5 +14,5 @@ hash-db = { path = "../../hash-db" , version = "0.15.2"}
 memory-db = { path = "../../memory-db", version = "0.31.0" }
 trie-root = { path = "../../trie-root", version = "0.17.0" }
 trie-db = { path = "../../trie-db", version = "0.24.0" }
-criterion = "0.3.3"
+criterion = "0.4.0"
```

Comment:
@ordian could you release a new version of `trie-bench` after merging this PR ?

Reply:
@koushiro just did publish it, thanks for the PR.